### DISCUSSION TITLE
Force trailer covers open

### DISCUSSION THREAD

**Discussion by nitroSkirata** [Courseplay/Courseplay_FS22#3556] [Q&A]
Hey,
is there a way, to disable the automatic closing of trailercovers by CP?
either a setting i am to blind to find, or a block of code to comment out?

I personally would love it, if the covers were kept open on the field.
This would help with manually unloading combines or for maneuvering around choppers.

**Comment by Tensuko** [comment]
Ok ok I will change that :)

Edit: Got to wait with that for FS25....
